src/components/Table/TableBody/TableBody.jsx

```diff
@@ -64,6 +64,7 @@ const propTypes = {
       isHidden: PropTypes.bool,
     })
   ).isRequired,
+  editBar: PropTypes.oneOf(['buttons', 'toast']),
```

Comment:
I don't understand why "toast" is an option here.  A toast is something that the application may show in response to the save (or even the cancel) action, but, I don't think the table itself will manage that.  @tay1orjones @davidicus am I wrong here?

Reply:
In my opinion, the toast showed up when I clicked save button according to the design. And the toast blocked the toolbar (i.e. when the toast shows up, the toolbar icons or save/cancel buttons are hidden ). That's why I think the toast belongs to the toolbar and is an option here.

Reply:
Does the design say that the toast should show up and cover the action bar?  That seems odd.  Something like a toast should be consistent within an application and not just random show anywhere on the screen.  I agree that the "story" needs to show this, I just think that the table needs to manage it.  How long does it stay on the screen, can I dismiss it, how do I translate it, how do i hook into undo (based on the message)?  Can I disable the "undo" (not every table in the world will support it) ... if you are going to manage a toast then you need to answer all these kinds of questions and provide solutions.  This is why I think that the application needs to manage the toast.  This can be demostrated in the story, but, not sure I agree that the table should be showing the toast.  I would expect that when I click save the edit toolbar is disabled (can't click it twice) or it goes away... but I would think there is a pattern for saving / pending save.  @tay1orjones @davidicus @JordanWSmith15

Reply:
@stuckless @tay1orjones  I would think that any toast is not managed by the table, but rather triggers from the product's implementation of a warning message (whatever that is), rather than a one-off toast built directly into the table. That way, the message format and location is consistent with all the other messages in the product.

Reply:
@karlinglee93 yeah, i see the PAL note has

> Display toast with undo option https://ibm.invisionapp.com/d/main#/console/15350624/319220019/preview (Use light version of toast. Dark version does not match PAL update)

But I do think that this is something that needs to be managed at the application level, but the story should show this behaviour so that someone can implement it.  ie, the Pattern says that a toast needs to be shown, and as such when implementing a table, the app will need to implement that pattern.  I just feel there is so much configuration around a toast that we'd end up with a huge 'toast' configuration node on the table just account for all this.

Reply:
@stuckless , I understood that, but I have no ideas about how to code when we don't manage the toast at the application level. ie, how the app will need to implement that pattern when implementing a table, how the toast showed up when I clicked save button and how we undid datas when I clicked the toast. I mean I just see one way we can solve the table editing features according to the design. Do you have some suggestions?